src/views/FormBuilder/components/Condition/Condition.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import { QuestionConditionTypes, QuestionTypes } from 'utils/helpers'
+import { QuestionConditionTypes, QuestionTypes } from 'utils/enums'
```

Comment:
data model should be in `models` directory, it has nothing to do with utillities